src/memary/agent/chat_agent.py

```diff
@@ -1,16 +1,14 @@
-import tiktoken
 from typing import Optional, List
 
 from memary.agent.base_agent import Agent
-from memary.agent.data_types import Context
 
 
 class ChatAgent(Agent):
 
     def __init__(self, name, memory_stream_json, entity_knowledge_store_json,
-                 system_persona_txt, user_persona_txt, past_chat_json):
+                 system_persona_txt, user_persona_txt, past_chat_json, llm_model_name="llama3", vision_model_name="llava"):
```

Comment:
Love this.
What about adding docstrings above `__init__` to inform what models we are able to use?